libs/wingsdk/src/target-sim/secret.inflight.ts

```diff
@@ -16,7 +15,8 @@ export class Secret implements ISecretClient, ISimulatorResourceInstance {
   private readonly name: string;
 
   constructor(props: SecretSchema) {
-    this.secretsFile = path.join(os.homedir(), ".wing", "secrets.json");
+    this.secretsFile = path.join(process.cwd(), ".env");
```

Comment:
This env file should already be loaded by the CLI and available in the preflight process.env
Also, it's worth noting that it may not only contain secrets.

Reply:
nvm this is the inflight file